tmt/utils.py

```diff
@@ -4846,13 +4850,31 @@ def distgit_download(
                 logger=logger)
 
 
+def _git_clone(
+        url: str,
+        destination: Path,
+        common: Common,
+        env: Optional[EnvironmentType] = None,
+        timeout: Optional[int] = None,
+        shallow: bool = False
+        ) -> CommandOutput:
+    """ Git clone url to destination """
+
+    depth = ['--depth=1'] if shallow else []
+    return common.run(Command('git', 'clone', *depth, url, destination), env=env, timeout=timeout)
+
+
 def git_clone(
         url: str,
         destination: Path,
         common: Common,
```

Comment:
Seems that the `common` object was handed over because previously we did not pass `logger`. So I guess we could now directly use `Commnad.run(..., logger=...)` and drop `common` altogether. @happz, would you agree?

Reply:
Ok, removed `common` in c7534a78 together with a bunch of other adjustments.